Spec/RealtimeClientChannel.swift

```diff
@@ -3423,6 +3423,40 @@ class RealtimeClientChannel: QuickSpec {
                 }
             }
 
+            // https://github.com/ably/ably-ios/issues/614
+            fit("should not crash when an ATTACH request is responded with a DETACHED") {
```

Comment:
`fit` just to show that the test is passing.

Reply:
OK, just learned about f* and x*... Kind of ugly in a framework which is all about descriptive test suites.